nova_plugin/host_aggregate.py

```diff
@@ -142,3 +158,15 @@ def list_host_aggregates(nova_client, **kwargs):
         HOST_AGGREGATE_OPENSTACK_TYPE,
         host_aggregates_list
     )
+
+
+@operation
+@with_nova_client
+def add_hosts(nova_client, hosts, **kwargs):
```

Comment:
@kbijakowski why not include hosts in the `create` operation and clean all when `delete`?